resources/src/ssec_tutorials/setup.py

```diff
@@ -28,14 +28,22 @@
 ASTROPY_GITHUB_FILE = os.environ.get("ASTROPY_GITHUB_FILE", "astropy-github.jsonl")
 ASTROPY_GITHUB = TUTORIAL_CACHE / ASTROPY_GITHUB_FILE
 
-QDRANT_ZIP_FILE = os.environ.get("QDRANT_ZIP_FILE", "scipy_qdrant.zip")
+QDRANT_ZIP_FILE = os.environ.get("QDRANT_ZIP_FILE", "scipy_qdrant_100k.zip")
 QDRANT_FOLDER_NAME = os.environ.get("QDRANT_FOLDER_NAME", "scipy_qdrant")
 QDRANT_PATH = TUTORIAL_CACHE / QDRANT_FOLDER_NAME
 QDRANT_COLLECTION_NAME = "arxiv_astro-ph_abstracts_astropy_github_documentation"
 
 
+def _verify_tutorial_cache_exits():
+    # Check if TUTORIAL_CACHE exists
+    if not TUTORIAL_CACHE.exists():
+        TUTORIAL_CACHE.mkdir(parents=True, exist_ok=True)
```

Comment:
This is not needed has the code above will make the directory and it's parents by default.